packages/primitives/src/reatomLinkedList.ts

```diff
@@ -74,6 +74,8 @@ export interface LinkedListAtom<
    * but it is not recommended to use it for large (thousands elements) lists */
   map: Key extends never ? never : Atom<Map<State<Node[Key]>, LLNode<Node>>>
 
+  linkedList: (initState: Array<Node>) => LinkedList<LLNode<Node>>
```

Comment:
Weird naming, may be?
```suggestion
  initiate: (initState: Array<Node>) => LinkedList<LLNode<Node>>
```